src/olympia/addons/serializers.py

```diff
@@ -303,10 +303,14 @@ def to_representation(self, obj):
         data = super(AddonSerializer, self).to_representation(obj)
         if 'theme_data' in data and data['theme_data'] is None:
             data.pop('theme_data')
-        if 'homepage' in data:
-            data['homepage'] = self.outgoingify(data['homepage'])
-        if 'support_url' in data:
-            data['support_url'] = self.outgoingify(data['support_url'])
+        if 'wrap_outgoing_links' in self.context['request'].GET:
+            if 'homepage' in data:
+                data['homepage'] = self.outgoingify(data['homepage'])
+            if 'support_url' in data:
+                data['support_url'] = self.outgoingify(data['support_url'])
+            if 'contributions_url' in data:
+                data['contributions_url'] = self.outgoingify(
+                    data['contributions_url'])
```

Comment:
I think all that can be simplified and more abstracted in a for loop

```python
if 'wrap_outgoing_links' in self.context['request'].GET:
    for key in ('homepage', 'support_url', 'contributions_url'):
        data[key] = self.outgoingify(data[key])
```

especially we're really just overwriting the data.

Reply:
Yep that's a good point.